internal/impl/confluent/processor_schema_registry_decode.go

```diff
@@ -87,6 +87,57 @@ When raw_unions is set to true then the above union schema is decoded as the fol
 - a `+"`Foo` instance as `{...}`, where `{...}` indicates the JSON encoding of a `Foo`"+` instance.
 `).Optional(),
 				service.NewBoolField("preserve_logical_types").Description(`Whether logical types should be preserved or transformed back into their primitive type. By default, decimals are decoded as raw bytes and timestamps are decoded as plain integers. Setting this field to true keeps decimal types as numbers in bloblang and timestamps as time values.`).Default(false),
+				service.NewBoolField("translate_kafka_connect_types").Description(`Only valid if preserve_logical_types is true. This decodes various Kafka Connect types into their bloblang equivalents when not representable by standard logical types according to the Avro standard.
+
+Types that are currently translated:
+
+.Debezium Custom Temporal Types
+|===
+|Type Name |Bloblang Type |Description
+
+|io.debezium.time.Date
+|timestamp
+|Date without time (days since epoch)
+
+|io.debezium.time.Timestamp
+|timestamp
+|Timestamp without timezone (milliseconds since epoch)
+
+|io.debezium.time.MicroTimestamp
+|timestamp
+|Timestamp with microsecond precision
+
+|io.debezium.time.NanoTimestamp
+|timestamp
+|Timestamp with nanosecond precision
+
+|io.debezium.time.ZonedTime
+|timestamp
+|Time with timezone (ISO-8601 format)
```

Comment:
Should there be a case for this in `translateKafkaConnectValue`? Also, it might be worth saying `ISO-8601 format with nanosecond precision` (same for `ZonedTimestamp` and also in the error message: `expected valid ISO formatted`)

Reply:
Good catch! This type doesn't exist :)